samples/subsys/testsuite/pytest/shell/pytest/test_shell.py

```diff
@@ -5,14 +5,15 @@
 import time
 import logging
 
-from twister_harness import Device
+from twister_harness import DeviceAdapter, TwisterHarnessTimeoutException
 
 logger = logging.getLogger(__name__)
 
 
-def wait_for_message(dut: Device, message, timeout=20):
+def wait_for_message(dut: DeviceAdapter, message: str, timeout: float = 20) -> bool:
```

Comment:
These `wait_for...` helpers methods can be not removed, `readlines_until` can be used instead.